core/domain/suggestion_registry.py

```diff
@@ -347,6 +352,14 @@ def validate(self):
                     exp_domain.STATE_PROPERTY_CONTENT,
                     self.change.property_name))
 
+        # The language_code field is used for querying purposes. We do not want
```

Comment:
Just say: "Suggestions of this type do not have an associated language code, since they are not translation-related."

Reply:
I don't think so, since you're talking about the language code of the suggestions rather than the language code of the object they refer to, right?

Reply:
For questions, I believe it's the language code of the question that's being added (for question suggestions you're always adding a new question). So it's both?

Reply:
Oh sorry, I see. I think the language code here is meant to identify the translation language for the suggestions. Questions are assumed to have a base language of English and I think that that is fine for the time being.

Reply:
Sounds good!